funders/index.md

```diff
@@ -30,3 +30,125 @@ A full list is [here](https://en.wikipedia.org/wiki/Crown_Research_Institute).
 - [The Alfred P. Sloan Foundation](https://sloan.org/)
 - [Chan Zuckerberg Initiative](https://chanzuckerberg.com/)
 - [Ford Foundation](https://www.fordfoundation.org/)
+
+## Individual Funding Opportunities
+
+The below funding opportunities have been taken from the [open-funding](https://github.com/ralphtheninja/open-funding) github repository and are sub-set of funding specifically relevant to academia.
+
+### [`Protocol Labs Research`](https://github.com/protocol/research#private-retrieval)
```

Comment:
Let's move private crypto to its own section, or remove it entirely.

Reply:
@RichardLitt given they are now broken out into their own pages, I marked which funders were web3 related in the list. What do you think of that?